src/ports/postgres/modules/graph/graph_utils.py_in

```diff
@@ -71,6 +70,43 @@ def _grp_from_table(tbl, grp_list):
                        for i in grp_list])
 
 
+def validate_output_and_summary_tables(model_out_table, module_name,
```

Comment:
The docstring and errors look good. I suggest removing the exclamation (`!`) from end of error messages.